README.didact.md

```diff
@@ -42,6 +42,21 @@ access all Camel K features.
 
 *Status: unknown*{#kamel-requirements-status}
 
+**Knative installed on the OpenShift cluster**
+
+The cluster also needs to have Knative installed and working.
+
+[Check if the Knative is installed](didact://?commandId=vscode.didact.requirementCheck&text=kservice-project-check$$oc%20api-resources%20--api-group=serving.knative.dev$$kservice%2Cksvc&completion=Verified%20Knative%20services%20installation. "Verifies if Knative is installed"){.didact}
+
+*Status: unknown*{#kservice-project-check}
+
+**Local OpenSSL Installation**
+
+[Check if the OpenSSL is installed](didact://?commandId=vscode.didact.requirementCheck&text=openssl-project-check$$openssl%20version$$OpenSSL&completion=Verified%20OpenSSL%20installation. "Verifies OpenSSL installation"){.didact}
```

Comment:
You can bypass `openssl` with `oc create secret generic example-event-streaming-user-reporting --from-file config/application.properties`

Reply:
Awesome! I was looking at how to do this. Thanks!